lib/boxcars/vector_store/hnswlib/build_from_files.rb

```diff
@@ -22,7 +22,7 @@ def initialize(params)
           validate_params(@training_data_path, @index_file_path, split_chunk_size)
 
           @json_doc_file_path = absolute_json_doc_file_path(@index_file_path, params[:json_doc_file_path])
-          @force_rebuild = params[:force_rebuild] || true
+          @force_rebuild = params.key?(:force_rebuild) ? params[:force_rebuild] : true
```

Comment:
i will double check this with the fix pr https://github.com/jaigouk/boxcars/pull/2